app/views/candidate_interface/work_history/explanation/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :title, page_title('out_of_work') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @work_explanation_form, url: candidate_interface_work_history_explanation_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl">Work history</span>
+        <%= t('page_titles.out_of_work') %>
+      </h1>
+
+      <p class="govuk-label">
+        If you are returning to work for the first time in 5 or more years,
+        please give details. For example, explain your parental or caring
+        responsibilities (or any other reason why you haven’t been employed).
+      </p>
+
+      <p class="govuk-label">
+        You can also include relevant work history from
+        more than 5 years ago. Providers will be interested in skills and
+        experience you could transfer to a school setting.
+      </p>
+
+      <%= f.govuk_text_area :work_history_explanation, label: { text: t('application_form.work_history.explanation.label') }, max_words: 600 %>
```

Comment:
Our form doesn't quite match the prototype. In it the label is hidden and uses 20 rows. I believe we can easily do this using the form builder by passing is `hidden: true` and `rows: 20`.

See below:
![image](https://user-images.githubusercontent.com/42817036/68025829-bcde4f00-fca5-11e9-96b2-533d26867fb2.png)